### ISSUE TITLE
Add option to do PSI test with a device that matches origin's field data

### ISSUE THREAD

**Issue by gbrlmtrz** [GoogleChromeLabs/psi#129]
Hey everyone! We've came across a bit of an issue with PSI where the results of the lab tests divert more and more from CWV data on select origins.

We believe this is due to the fact that PSI uses a "synthetic" device that tries to be similar to the avg device on the open web, but it may not reflect the core audience of a given site, so we though it may be cool to have an option for PSI to do the test based on a device that properly reflects this audience

To give an example of the discrepancy, please take a look at the results for this origin, specially LCP (2.0 on cwv vs 8.7 on psi) https://hamuesgyemant.hu/ they are day and night!

![image](https://user-images.githubusercontent.com/12701019/130973281-21c9c01e-306b-41a9-b39a-abedd15756a6.png)

![image](https://user-images.githubusercontent.com/12701019/130973445-7d8a1646-957d-4c14-ba85-eb62d81d3ea3.png)

We think that if, for example, PSI would detect that most visitors for this origin come with a 4g connection and a Pixel 5 device (maybe using CRUX data) and do the test based on these parameters, then it would give a score that would do a better job at reflecting the status of the page for its actual users. And it would also turn PSI into a much more valuable tool for debugging because as it stands right now what is does is scare and confuse non-technical people into thinking their site is performing poorly and frustate technical people when they know that their core or target audience comes from devices that are well better than PSIs defaults (a moto g4 and a 3g connection iirc)

**Comment by alonkochba** [comment]
I agree, any site with field data that is mostly 4G+, will have much better field data than lab data in Mobile, by design. There are countless examples for such sites.

Most of the world uses 4G, the main frustration comes not from advanced users who understand throttling and field data, but from users who see a bad score and don't understand why.
Even worse, most sites will never even have enough sessions to have field data in order to know the difference.

It would be great to at least see an option to run Mobile on 3G **or** 4G throttling, and make this more clear.
If the user has enough field data, it would be even nicer to default to the more common connection, as @gbrlmtrz suggested.

While we're at it - if there's field data - why not display a score according to field data when it exists, to highlight the importance of field data?
It's not an intuitive order to have Lab Score, Field Data, Lab Data - people misrepresent the score as the most important part even if the site has green CWV.
